File: Test1_solution/solution_11.java
public class solution_11
{
    public static void main(String args[])
    {
        int arr[]={2,2,1,1,2};
        int count=0;
        int ele=0;
        for(int i=0;i<arr.length;i++)
        {
            if(count==0)
            {
                ele=arr[i];
            }
            count+=(ele==arr[i])?1:-1;
        }
        System.out.println("majority ele="+ele);
    }
}

File: Test1_solution/solution_11.py
def majority_ele(nums,n):
    count=0
    ele=0
    for i in range(0,n):
        if count==0:
            ele=nums[i]
        count+=1 if (ele==nums[i]) else -1
    return ele
# nums = [3,2,3]
nums=[2,2,1,1,2]
majority_ele=majority_ele(nums,len(nums))
print("Majority_ele=",majority_ele)